subsys/regulator/Kconfig

```diff
@@ -0,0 +1,33 @@
+menuconfig REGULATOR
+	bool "Regulator"
+	help
+	  This option enables regulator support
+
+if REGULATOR
+
+menuconfig REGULATOR_PI_REG
+	bool "PI Regulator"
+	help
+	  Enable the PI regulator module
+
+if REGULATOR_PI_REG
+
+config REGULATOR_PI_REG_BT_MESH
```

Comment:
It looks strange that this is under REGULATOR_PI_REG_BT_MESH. Could this be under LIGHT_CTRL instead and dependent on REGULATOR and REGULATOR_PI_REG. It will be hard to find to activate.
Or Could this be on same level as the REGULATOR and then depend on the other two. It is just that it is 3 levels of menu hierarchy.